code/obj/submachine/seed.dm

```diff
@@ -592,6 +592,8 @@
 				DNA.endurance = SpliceMK2(P1DNA.alleles[7],P2DNA.alleles[7],P1DNA.vars["endurance"],P2DNA.vars["endurance"])
 
 				boutput(usr, "<span class='notice'>Splice successful.</span>")
+				//0 xp for a 100% splice, 4 xp for a 10% splice
+				JOB_XP(usr, "Botanist", round((100 - splice_chance) / 20))
```

Comment:
I'd just clamp the max/min this could be for safety since it's on a constant value.

Reply:
Added a clamp between 0 and 4 xp.

Reply:
wait fuck

Reply:
fixed now